react/week4/src/components/UsersFetch.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from "react";
+import { UserProvider } from './UserContext';
+import UsersSearch from './UsersSearch';
+import UsersList from './UsersList';
+
+export default function UsersFetch() {
+    const [users, setUsers] = useState("")
+    const [result, setResult] = useState([{ id: 1, login: "No result" }]);
```

Comment:
I would say this should be an array
```suggestion
    const [result, setResult] = useState([]);
```